### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by zhaojj2209**
Update user guide to reflect commands for v1.2. The UI image has also been updated with an actual image of the current UI.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,118 +14,185 @@ Fine\$$e is a **desktop app for managing finances, optimized for use via a Comma
...
 
-Format: `add-income d/DESCRIPTION a/AMOUNT`
+* `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
+* `DATE` should be in `dd/mm/yyyy` format.
 
 Examples:
-- `add-income d/Internship a/1200` will add a new income with description “Internship” and amount $1200.00.
-- `addi d/Random Angpao a/10` will add a new income with description “Random Angpao” and amount $10.00.
+* `add-income t/Internship a/560 d/03/10/2020 c/Work`
+* `addi t/Angpao money a/$20 d/10/10/2020 c/CNY c/Gift`
```
Comment (ianyong) [inline docs/UserGuide.md:86]: Should the date be set in the CNY period?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,118 +14,185 @@ Fine\$$e is a **desktop app for managing finances, optimized for use via a Comma
...
 
-Format: `rm-income INDEX` / `rmi INDEX`
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
- - Deletes the specified income entry from the finance tracker.
- - The index refers to the index number shown in the displayed income list.
- - The index **must be a positive integer** 1, 2, 3, ...
+* On Overview tab: searches all transactions (expenses/incomes).
+* On Expenses tab: searches all expenses.
+* On Income tab: searches all incomes.
```
Comment (ianyong) [inline docs/UserGuide.md:106]: Perhaps bullet points detailing the behaviour of the command on each tab could be moved from the format section to the description section? @yongping827 what do you think?

**Review by ianyong — COMMENTED** [review]
@zhaojj2209 Thanks for updating the user guide!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -14,118 +14,185 @@ Fine\$$e is a **desktop app for managing finances, optimized for use via a Comma
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `finesse.jar` from [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
+1. Download the latest `fine$$e.jar` from [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
```
Comment (wltan) [inline docs/UserGuide.md:17]:
```suggestion
1. Download the latest `fine$$e.jar` from [here](https://github.com/AY2021S1-CS2103T-W16-3/tp/releases).
```

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by yongping827 — APPROVED** [review]
LGTM

**Review by wltan — APPROVED** [review]
LGTM

**Pull request merged**